scripts/system/create/entityProperties/html/js/entityProperties.js

```diff
@@ -113,11 +116,16 @@ const GROUPS = [
                 },
                 propertyID: "primitiveMode",
             },
+            {
+                label: "Render With Zones",
+                type: "multipleZonesSelection",
+                propertyID: "renderWithZones",
+            }
```

Comment:
This is too early to include? (This property is added in https://github.com/kasenvr/project-athena/pull/78 which hasn't been merged yet?)
The UI control displays but appears not to be usable, so may need to be removed in this PR.

Reply:
I lost the original version of that file for that specific PR done sadly directly on the core branch
I fear a regression. 
I will comment this property for now.